### PR TITLE
[CW-3646] Added support for enumDisabled uiSchema option in SelectWidget and ArrayField.

### PR DESCRIPTION
**Pull request by michal-damiecki**
#### What has changed in the code ####

Added support for `enumDisabled` `uiSchema` option in `SelectWidget` and `ArrayField`.

Peek 2020-12-22 10-05

#### Reasons for making this change ####
    
UI should disable reserved tags.

### Checklist

* [X] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated docs if needed
* [ ] **I'm adding a new feature**
  - [ ] I've added unit test for feature
  - [ ] I've ran lint

### CODE REVIEW

File: src/Form/formComponents/fields/ArrayField/ArrayField.js [hunk 1]
```diff
@@ -363,12 +363,16 @@ export default class ArrayField extends Component {
               const itemIdPrefix = idSchema.$id + '_' + index;
               const itemIdSchema = toIdSchema(itemsSchema, itemIdPrefix, definitions);
 
+              const isReserved = uiSchema &&
```
Comment (stepek) [inline src/Form/formComponents/fields/ArrayField/ArrayField.js:366]: `isDisabled`

File: src/components/forms/Select/Select.js [hunk 2]
```diff
@@ -189,25 +195,31 @@ class Select extends Component {
                 }
                 {options.map((item, i) => {
                   if (typeof item === 'object') {
+                    const isReserved = enumDisabled.includes(item.label);
```
Comment (stepek) [inline src/components/forms/Select/Select.js:198]: isDisabled

**Review by stepek — CHANGES_REQUESTED** [review]

File: src/Form/formComponents/fields/ArrayField/ArrayField.js (hunk 1 above)
Comment (michal-damiecki) [reply src/Form/formComponents/fields/ArrayField/ArrayField.js]: @stepek Fixed.

File: src/components/forms/Select/Select.js (hunk 2 above)
Comment (michal-damiecki) [reply src/components/forms/Select/Select.js]: @stepek Fixed.

**Review by stepek — APPROVED** [review]

**Pull request merged**